packages/jellyfish-transaction/src/script/crypto.ts

```diff
@@ -18,6 +18,15 @@ export class OP_SHA1 extends StaticCode {
   }
 }
 
+/**
+ * The input is hashed using SHA-3.
+ */
+export class OP_SHA3 extends StaticCode {
```

Comment:
Is this being used by the `evmtx` internally? ain script [here](https://github.com/DeFiCh/ain/blob/cb38e0fc35920e34b2e8451128b3724d590ffd59/src/script/script.h#L167), just want to make sure if it is needed

Reply:
Nope, not used by evmtx. just thought it would be nice to keep jellyfish updated

Reply:
@canonbrother can we keep this `OP_SHA3` here? Or should I remove?

Reply:
hmm... i think its fine to keep there